<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bremswegrechner</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="labelFahrzeugauswahl">
        <property name="text">
         <string>Bitte Fahrzeug auswählen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxGefahrenbremsung">
        <property name="text">
         <string>Gefahrenbremsung</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelUntergrundauswahl">
        <property name="text">
         <string>Bitte Untergrund auswählen</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QComboBox" name="comboBoxFahrzeug"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxUntergrund"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelGeschwindigkeitseingabe">
        <property name="text">
         <string>Bitte Geschwindigkeit in km/h eingeben</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditGeschwindigkeit">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButtonBremswegberechnen">
        <property name="text">
         <string>Bremsweg berechnen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdBremsweg"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
